### ISSUE TITLE
Undefined method `match?' for nil:NilClass when processing response without content type

### ISSUE THREAD

**Issue by kakuzei** [soberstadt/faraday-decode_xml#3]
Hello,
I'm currently encountering an issue when there is no content type available in the response of the http request:

```
Error: undefined method `match?' for nil:NilClass

          pattern.is_a?(Regexp) ? type.match?(pattern) : type == pattern
                                      ^^^^^^^
.../faraday-decode_xml-0.2.0/lib/faraday/decode_xml/middleware.rb:41:in `block in process_response_type?'
```

The issue is due to the `response_type` method which returns `nil` instead of an empty string:
```
def response_type(env)
  env.response_headers["Content-Type"].to_s.split(";", 2).first
end
```

Regards

**Comment by soberstadt** [comment]
Fixed with 0.2.1 https://rubygems.org/gems/faraday-decode_xml/versions/0.2.1

**Comment by kakuzei** [comment]
Thank you for the quick fix!
